relationships/synthesis/APM-APPLICATION-to-INFRA-ELASTICACHEREDIS.yml

```diff
@@ -0,0 +1,44 @@
+relationships:
+  - name: apmServiceCallsInfraElastiCacheRedisCluster
+    version: "1"
+    origins:
+      - APM Metrics
+    conditions:
+      - attribute: metricName
+        regex: "^Datastore/instance/Redis/[^/]*\\.cache\\.amazonaws\\.com/[0-9]+"
+    relationship:
+      expires: P75M
+      relationshipType: CALLS
+      source:
+        extractGuid:
+          attribute: entity.guid
+      target:
+        lookupGuid:
+          candidateCategory: REDIS
+          fields:
+            - field: endpoint
+              attribute: metricName__4
+            - field: port
+              attribute: metricName__5
+
+  - name: apmServiceCallsInfraElastiCacheRedisNode
```

Comment:
Aren't these the exact same rules? Is there isn't any difference, we only need one.

Reply:
@javimb There's a slight change in regex which i have update, please review.